apps/projects/app/components/Shared/FilterBar/FilterBar.js

```diff
@@ -156,7 +179,12 @@ class FilterBar extends React.Component {
           </FilterDropDown>
 
           <FilterDropDown caption="Labels" enabled={ Object.keys(filtersData.labels).length > 0}>
-            { Object.keys(filtersData.labels).map(
+            { Object.keys(filtersData.labels).sort(
```

Comment:
This should be calculated outside the return and captured into a variable instead

Reply:
Discussed, left for refactoring in the nearest future.